### DISCUSSION TITLE
The impact of the antenna.ctl control file

### DISCUSSION THREAD

**Discussion by vlbi-jun-yang** [nvi-inc/fs#154] [Q&A] [answered]
There are multiple telescopes to run VLBI observations at Onsala. I was told to re-link the file antenna.ctl to the correct antenna control file before running the VLBI observations.  At Onsala, the 20-m and 25-m antennas were not directly under the control of FS. So, the control file may not be really used. Is my understanding correct here?

**Comment by wehimwich** [comment, marked as answer]
The _antenna.ctl_ control file is _mostly_ just informational, but not entirely. It contains information about the antenna, specifically: diameter, slew rates, limits, and axis type. These parameters are recorded in the header each time a log is opened.

I can't say how your local _antcn_ program uses the _antenna.ctl_ parameters, but I believe the only one used in the FS at this time is the diameter. This is used by the `fivept` and `onoff` commands to estimate the FWHM of the beam if the active _.rxg_ control files specify a `frequency` dependence for the FWHM. In turn, the FWHM is used to calculate the step size according to the settings provided in the commands. The FWHM is also used with the crude FS source models to, inversely proportionally, adjust the flux for source resolution.

Beyond any effect on your _antcn_, the net impact of not having the correct _antenna.ctl_ file is only for antenna calibration runs where it will affect how far off source the antenna is moved and the flux used in the SEFD and Tcal(j) calculations. This will also affect the value of the Tcal(r), used as a quality check of the current calibration.

**Comment by vlbi-jun-yang** [comment]
Thanks for the answer in details. It is clear for me now.
